hrms/hr/doctype/full_and_final_statement/full_and_final_statement.py

```diff
@@ -309,6 +309,11 @@ def get_account_and_amount(ref_doctype, ref_document):
 		amount = details.paid_amount - (details.claimed_amount + details.return_amount)
 		return [payment_account, amount]
 
+	if ref_doctype == "Leave Encashment":
```

Comment:
Leave Encashment creates an additional salary on submission so its always paid via salary slip. How to pay Leave Encashment is not optional currently.

We will have to set `paid_via_salary_slip` to True for Leave Encashment items in FnF so that they are not considered again in total payable - will lead to double payment

Reply:
Leave Encashment currently does not have a flow to pay separately via Payment Entry/JV. It _always_ creates an additional salary on submission. This would be a separate feature where encashment will have to give an option on how to pay - via salary or creating a separate payment entry. In that case, only the latter could be considered in FnF. 

But this feature should be added via a separate PR. As per the current workflow, paid via salary slip will have to be set to True for encashment.

Reply:
To be handled separately